recipes/osx-sysroot/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "macosx-sysroot" %}
+{% set version = "1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+
+build:
+  noarch: generic
```

Comment:
I think you'll need that per output?

Reply:
yeah you do! TIL! will fix and thank you!